decidim-dev/lib/decidim/dev/test/w3c_rspec_validators_overrides.rb

```diff
@@ -1,5 +1,38 @@
 # frozen_string_literal: true
 
+# This is a temporary fix to ignore some HTML/CSS validation issues with the
+# Decidim HTML validation process.
+#
+# See: https://github.com/decidim/decidim/pull/10014
+# Related: https://github.com/rails/rails/issues/46405
+module W3CValidators
+  class NuValidator
+    protected
+
+    alias validate_nu validate unless method_defined?(:validate_nu)
+
+    def validate(options) # :nodoc:
+      filter_results(validate_nu(options))
+    end
+
+    def ignore_errors
+      @ignore_errors ||= [
+        "An “input” element with a “type” attribute whose value is “hidden” must not have an “autocomplete” attribute whose value is “on” or “off”."
```

Comment:
Great work 🚀!
Now there's [another rule that's failing](https://github.com/decidim/decidim/actions/runs/3376192642/jobs/5603630886) and that we should also ignore: 

>     line 662: An “input” element with a “type” attribute whose value is “hidden” must not have any “aria-*” attributes.

Can you add it 🙏🏽 ? Thanks

Reply:
I'm just investigating it...

Locally on my machine I cannot replicate that error. Just trying to figure out if this is coming from Decidim JS or from Foundation (so that we can fix the root problem).

Reply:
Fixing the foundation issue in another PR foundation/foundation-sites#12496.

While we wait for a new Foundation release, let's add that to the ignored errors.

Reply:
Added the second ignore at 40ad75658b6fb59207c578e99adf14a37ff87043.